.github/workflows/.windows.yml

```diff
@@ -337,7 +337,7 @@ jobs:
           }
       -
         name: Starting containerd
-        if: matrix.runtime == 'containerd'
+        if: matrix.runtime == 'containerd' && matrix.os == 'windows-2019'
```

Comment:
Does this apply to all `run`'s below?

Reply:
This step to "Start Containerd" manually has been removed in favour of the Docker-managed instance.